test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/CommunityVocabularyTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CommunityVocabularyTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Text;
+using Microsoft.OData.Edm.Csdl;
+using Microsoft.OData.Edm.Validation;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OData.Edm.Vocabularies.Community.V1;
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.Vocabularies
+{
+    public class CommunityVocabularyTests

```

Comment:
>CommunityVocabularyTests [](start = 17, length = 24)

Maybe add test cases:

1. In the CsdlWriter, create the model with function decorated with this term?

2. In the CsdlReader, read the csdl with function decorated with this term.

Reply:
Yes these tests will be added, as part of EDM model validation test along with bound function, in the task for upper layer and as a separate PR. This PR here is just to add the CommunityVocabulary.